circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_ordering.cpp

```diff
@@ -156,6 +156,8 @@ void match_nullifiers_to_commitments_and_squash(
                 // match found!
                 // squash both the nullifier and the commitment
                 // (set to 0 here and then rearrange array after loop)
+                info("important: chopped commitment for siloed inner hash note \n" +
```

Comment:
I don't understand this error message tbh - and I doubt external users will too. For my understanding - if someone sees this, what should they do? What's the real error? Is it even an error? (I assume not because we aren't using `abort:`

Reply:
@rahul-kothari @suyash67 I think Suyash forgot to remove a cherry-picked commit of another PR (#1643).

@rahul-kothari This log is an information which is used to provide context in case another error is thrown. Should we rename 'important' flag through 'information'? Please have a look at #1643 and you can comment there.

Reply:
Removed the cherry-picked commit from your branch @jeanmon